README.md

```diff
@@ -11,43 +11,189 @@ description: Creates an Azure AI Service and deploys the specified models.
-By default this template will use a default naming convention to prevent naming collisions within Azure.
-To override default naming conventions the following can be set.
+(Embed demo video here)
 
-- `AZURE_OPENAI_NAME` - The name of the Azure Open AI service
+## Getting Started
 
-Run `azd config set <key> <value>` after initializing the template to override the resource names
+You have a few options for getting started with this template. The quickest way to get started is [GitHub Codespaces](#github-codespaces), since it will setup all the tools for you, but you can also [set it up locally](#local-environment). You can also use a [VS Code dev container](#vs-code-dev-containers)
 
-## Reporting Issues and Feedback
+This template uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
 
-If you have any feature requests, issues, or areas for improvement, please [file an issue](https://aka.ms/azure-dev/issues). To keep up-to-date, ask questions, or share suggestions, join our [GitHub Discussions](https://aka.ms/azure-dev/discussions). You may also contact us via AzDevTeam@microsoft.com.
+  * We recommend using [SUGGESTED REGION]
+
+### GitHub Codespaces
+
+You can run this template virtually by using GitHub Codespaces. The button will open a web-based VS Code instance in your browser:
+
+1. Open the template (this may take several minutes)
+    [![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](placeholder)
+2. Open a terminal window
+3. Sign into your Azure account:
+
+    ```shell
+     azd auth login --use-device-code
+    ```
+
+4. [any other steps needed for your template]
+5. Provision the Azure resources and deploy your code:
+
+    ```shell
+    azd up
+    ```
+
+6. (Add steps to start up the sample app)
+
+### VS Code Dev Containers
+
+A related option is VS Code Dev Containers, which will open the project in your local VS Code using the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers):
+
+1. Start Docker Desktop (install it if not already installed)
+2. Open the project:
+    [![Open in Dev Containers](https://img.shields.io/static/v1?style=for-the-badge&label=Dev%20Containers&message=Open&color=blue&logo=visualstudiocode)](placeholder)
+3. In the VS Code window that opens, once the project files show up (this may take several minutes), open a terminal window.
+4. Sign into your Azure account:
+
+    ```shell
+     azd auth login
+    ```
+
+5. [any other steps needed for your template]
+6. Provision the Azure resources and deploy your code:
+
+    ```shell
+    azd up
+    ```
+
+7. (Add steps to start up the sample app)
+
+8. Configure a CI/CD pipeline:
+
+    ```shell
+    azd pipeline config
+    ```
+
+### Local Environment
```

Comment:
shoudn't devcontainer be one of local environment?

Reply:
I am a bit confused by this question since .devcontainers is listed as an option https://github.com/Azure-Samples/azd-ai-starter/blob/7e2eb3cc13e03be8d47f49ea09f363fabba390c1/README.md#vs-code-dev-containers